pages/home.py

```diff
@@ -29,6 +31,14 @@ def is_task_in_progress(self):
     def task_in_progress(self):
         return self.selenium.find_element(*self._task_in_progress_locator).text
 
+    @property
+    def displayed_suggested_first_tasks_list(self):
+        return self.is_element_visible(self._displayed_suggested_first_tasks_list_locator)
+
+    @property
+    def suggested_first_tasks_list(self):
```

Comment:
We try not to create properties that return a bunch of elements from the page (which is what this would do). We prefer to return objects. I suggest doing something similar to what we've done with `available_tasks` at https://github.com/mozilla/oneanddone-tests/blob/master/pages/tasks/available_tasks.py#L20. Note that in order to do that you'll also need to include the code for the `Task` class, which begins at https://github.com/mozilla/oneanddone-tests/blob/master/pages/tasks/available_tasks.py#L28.

This will cause some code duplication which ideally we'd find a way to refactor, but don't worry about that yet. Just copying the code for the `Task` class into this page object for now is fine.